libraries/lts/include/mcrl2/lts/detail/liblts_bisim_m.h

```diff
@@ -0,0 +1,579 @@
+// Author(s): Jan Martens
```

Comment:
I would probably name this liblts_bisim_minimal or something (maybe even _minimal_depth) for more clarity.